triggers/k8s/airflow-operator.yaml

```diff
@@ -21,12 +21,11 @@ github:
   pullRequest:
     branch: .*
     commentControl: COMMENTS_ENABLED
-id: e84b1610-5061-49af-b8e0-62665d2369c5
```

Comment:
https://github.com/GoogleCloudPlatform/click-to-deploy/pull/923/files#r405875201

Reply:
I had to close the other PR because the CB triggers are not working properly. I am ok in keeping the codebase on sync with the created builders.